src/excmds.ts

```diff
@@ -3266,7 +3266,8 @@ export async function taball(id: string) {
  * entering `:tn 43` will expand to `:tabnext_gt 43`.
```

Comment:
I think this could be a bit clearer:

1. `js [...]` is still and ex-command, this makes it sound like it isn't
2. the key problem with #3026 is that it wasn't clear that `command`s can accept arguments - I think really what we need to do is add a bit to this to say something along the lines of "if you need to control where the argument goes in the ex-command, use [[js]] with the -d or -p options"

Thanks for taking the time to look at this. What do you think? : )

Reply:
For clarity the solution that saulrh couldn't find in #3026 was `command loudecho js -d€ window.alert(JS_ARGS.join(" "))€`.

Reply:
Damn, I may have managed to completely miss the point of the issue... so if *now* I understand correctly, the desired effect was to create an alias with composite that passes the arguments to any part of the composite command other than the last, i. e. something like `composite <command> <arguments go here> | <command>`?

Reply:
Not quite.

There's three levels to this:

An ex-alias is a shorthand for a longer ex-command, e.g.

```
:command top scrollto 0
:top # takes you to the top of the page
```

An ex-alias lets you rename an ex-command:

```
:command t tabopen
:t news.bbc.co.uk # opens BBC News
```

An ex-alias lets you create a new ex-command:

```
:command tabclosethenrecent composite tabclose ; tab #
:tabclosethenrecent # closes your tab then takes you to the most recent one
```

Under the hood `:command` works exactly the same in all cases. #3026 is asking how to control where the arguments given (like in the second case) go.

For example say I have 

```
scrollto x y
```

and I want to make a new command

`otllorcs y x` which is the does the same as `scrollto x y`.

An answer is

```
:command otllorcs js -d€ tri.excmds.scrollto(JS_ARG[1],JS_ARG[0])€
:scrollto 100 0
:otllorcs 0 100
```

Is that clearer? It's not really anything to do with composite but it could be used with it.

Reply:
I think I've got it now. Let's see if I can think of a good way of putting it into the doc.

Reply:
Nope. *I* know exactly what I mean, but I can't think of any good wording.